ts/components/AbortOnboardingModal.tsx

```diff
@@ -21,11 +26,11 @@ class AbortOnboardingModal extends React.PureComponent<Props> {
     return (
       <Modal visible={true} onRequestClose={this.props.onClose}>
         <Container>
-          <AppHeader>
-            <Right>
-              <TouchableHighlight onPress={this.props.onClose}>
+          <AppHeader noLeft={true}>
+            <Right style={styles.rightOnly}>
```

Comment:
I'm not sure about this, why use the `Right` component if it doesn't work here? shouldn't we use a simple `View`? what do you think? - cc @fpersico

Reply:
Fixed <Right> behaviour. A style property was necessary due to a wrong setting from Headers.ts